azext_edge/edge/providers/support/base.py

```diff
@@ -180,164 +180,83 @@ def process_deployments(
 
 def process_statefulset(
     resource_api: EdgeResourceApi,
-    label_selector: str = None,
-    field_selector: str = None,
-):
-    from kubernetes.client.models import V1StatefulSet, V1StatefulSetList
-
+    child_group: str = "",
+    field_selector: Optional[str] = None,
+    label_selector: Optional[str] = None,
+) -> List[dict]:
     v1_apps = client.AppsV1Api()
 
-    processed = []
-
-    statefulsets: V1StatefulSetList = v1_apps.list_stateful_set_for_all_namespaces(
-        label_selector=label_selector, field_selector=field_selector
+    return _process_kubernetes_resources(
+        child_group=child_group,
+        resources=v1_apps.list_stateful_set_for_all_namespaces(
+            label_selector=label_selector,
+            field_selector=field_selector
+        ),
+        resource_api=resource_api,
+        kind="Statefulset",
```

Comment:
Should we use an enum or alternative mapping ?

Reply:
if an enum that is close enough exists, (ex: has spaces or is all lowercase) - use that over creating and managing a new one

Reply:
hmm I don't see an exist one that's close, will think of a new one